docs/DeveloperGuide.md

```diff
@@ -434,12 +434,194 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 (For all use cases below, the **System** is the `ProfBook` and the **Professor** is the `user`, unless specified
 otherwise)
 
-**Use case: Delete a person**
+**Use case: help**
 
 **MSS**
 
-1. User requests to delete a student/group/task with specific id for each one
-2. AddressBook deletes the person
+1. User requests for help.
+2. ProfBook displays help message, informs user how to input help with additional information specific to command.
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. User inputs help with specific command.
+
+    * 2a1. ProfBook displays help message specific to input command
+
+      Use case ends.
+
+
+* 2b. User inputs invalid command together with help.
+
+    * 2b1. ProfBook shows an error message.
+
+      Use case resumes at step 1.
+
+**Use case: Cd into group**
+
+**MSS**
+
+1. User requests to cd into a specific group
+2. ProfBook moves into the directory, display updates to show students within specified group
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. The path of the group is invalid
+
+    * 2a1. ProfBook shows an error message.
+
+      Use case resumes at step 1.
+  
+
+* 2b. The path of the group is empty
+  
+  * 2b1. ProfBook shows an error message.
+
+      Use case resumes at step 1.
+
+**Use case: ls to view children**
+
+**MSS**
+
+1. User requests to view children either within current directory or specified path.
+2. ProfBook displays list of children.
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. Provided path is invalid
+
+    * 2a1. ProfBook shows an error message.
+
+      Use case resumes at step 1.
+
+* 2b. Command has no arguments
+
+    * 2b1. ProfBook shows list of children under current directory.
+
+      Use case ends.
+
+**Use case: View tasks under a specified path**
+
+**MSS**
+
+1. User requests to view tasks under specified path, either student or group.
```

Comment:
User is able to use the cat command with and without specifying a path. Perhaps could add more details like the use case for ls above.

Reply:
Added a extension for this thank you